resources/js/features/user-game-activity/components/UserGameActivityTimeline/ClientLabel/ClientLabel.tsx

```diff
@@ -0,0 +1,72 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuMonitor } from 'react-icons/lu';
+
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+import { cn } from '@/common/utils/cn';
+import { buildEmulatorClientLabel } from '@/features/user-game-activity/utils/buildEmulatorClientLabel';
+
+import { ActivityBasicLabel } from '../../ActivityBasicLabel';
+import { ActivityTooltipWrapper } from '../../ActivityTooltipWrapper';
+import { ManualUnlockLabel } from './ManualUnlockLabel';
+
+interface ClientLabelProps {
+  session: App.Platform.Data.PlayerGameActivitySession;
+}
+
+export const ClientLabel: FC<ClientLabelProps> = ({ session }) => {
+  const { t } = useTranslation();
+
+  if (session.type === 'reconstructed') {
+    return (
+      <ActivityTooltipWrapper
+        Icon={LuMonitor}
+        label={t('Reconstructed Session')}
+        t_tooltip={t(
+          'This session is missing emulator, playtime, and rich presence information. The session was automatically reconstructed by the server with what metadata we have.',
+        )}
+        className="text-neutral-500 light:text-neutral-400"
+      />
+    );
+  }
+
+  if (session.type === 'manual-unlock' && session.events?.[0].unlocker) {
+    return <ManualUnlockLabel unlocker={session.events[0].unlocker} />;
+  }
```

Comment:
Manual unlocks don't appear in the middle of a valid session:
![image](https://github.com/user-attachments/assets/c875d64c-2b33-4f68-a1ff-1a7ed1372ce8)
![image](https://github.com/user-attachments/assets/ea162425-2b19-4c28-b631-25cef5604f91)

Outside a valid session:
![image](https://github.com/user-attachments/assets/d4f34077-ea2a-4888-a560-7be304b5f026)

Reply:
Resolved in latest:
![Screenshot 2025-01-02 at 6 38 16 PM](https://github.com/user-attachments/assets/f1a908bc-4b59-4b4a-ab2f-def013208c23)